.gitlab/issue_templates/general.md

```diff
@@ -0,0 +1,33 @@
+Overview:
+----------------------------------------
+_Please describe your issue in detail._
+
+Reproduction steps:
+----------------------------------------
+1. Click on **Contacts -> New Individual**.
+1. Entered **First Name** and **Last Name** and clicked **Save**.
+1. Got an error "**Fatal error: DB error**".
+
+````(Please include the full text of the error message. You can remove any private information like passwords. If you put it inside three backtick ` symbols it will wrap/format it nicely!)````
+
+Current behaviour:
+----------------------------------------
+_What happens currently. Please provide screenshots or gifs ([LICEcap](http://www.cockos.com/licecap/), [SilentCast](https://github.com/colinkeenan/silentcast)) where appropriate._
+
+Expected behaviour:
+----------------------------------------
+_What should happen. Please provide screenshots or gifs ([LICEcap](http://www.cockos.com/licecap/), [SilentCast](https://github.com/colinkeenan/silentcast)) where appropriate._
+
+Environment information:
+----------------------------------------
+
+* __Web Server:__ _Apache/LiteSpeed/Nginx/Other_  
+* __CiviCRM Version:__ _Master_
+* __CMS:__ _Backdrop/Drupal/Joomla!/WordPress_
+* __CMS Version:__ _1.0_
+* __Database:__ _MariaDB/MySQL/Percona/Other_
+* __Database Versions:__ _10.4_
```

Comment:
Some of these have lists that you can choose from, and some are just one of many possible values. I think it might be a little clearer that you need to fill every item if either... (a) all the fields had multiple examples

```
* __Web Server:__ _Apache/LiteSpeed/Nginx/..._  
* __CiviCRM Version:__ _Master/5.20.0/5.19.1/5.18.2/..._
* __CMS:__ _Backdrop/Drupal/Joomla/WordPress_
* __CMS Version:__ _1.0/4.5/6.10/7.30/..._
* __Database:__ _MariaDB/MySQL/Percona/Other_
* __Database Versions:__ _5.5/5.7/10.3/10.4/..._
```

Or all the fields had FIXME:

```
* __Web Server:__ _FIXME (Ex: Apache, Nginx)_ 
* __CiviCRM Version:__ _FIXME (Ex: 5.19.1)_
* __CMS:__ _FIXME (Ex: WordPress)_
* __CMS Version:__ _FIXME (Ex: 4.5)_
* __Database:__ _FIXME (Ex: MySQL, MariaDB)_
* __Database Versions:__ _FIXME (Ex: 5.7.7)_
```

Reply:
Capturing the user's "Browser" is also helpful with bug reports.